core/snapshot_util.rs

```diff
@@ -35,11 +37,12 @@ pub fn create_snapshot(create_snapshot_options: CreateSnapshotOptions) {
   let snapshot = js_runtime.snapshot();
   let snapshot_slice: &[u8] = &snapshot;
   println!(
-    "Snapshot size: {}, took {}s ({})",
+    "Snapshot size: {}, took {:#?} ({})",
     snapshot_slice.len(),
-    start.elapsed().as_secs_f64(),
+    Instant::now().saturating_duration_since(mark),
```

Comment:
Is this not the same thing?

Reply:
I want to measure how long each step takes so I need to reset the timer after each step